app/Models/Post.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Post extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    protected $fillable = ['subject', 'content', 'author_id', 'team_id', 'challenge_id', 'banner_id'];
```

Comment:
Can be replaced with `guarded` (which is the opposite of `fillable`):
```suggestion
    protected $guarded = ['id'];
```

Reply:
oh that's great